### PR TITLE
Gerichtsfinder JSON

### PR DESCRIPTION
**Pull request by chohner**

### CODE REVIEW

File: app/services/gerichtsfinder/updateFromS3.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import fs from "node:fs";
```
[the remark this answers is not in the document]
Comment (chohner) [reply app/services/gerichtsfinder/updateFromS3.ts:1]: Yea I also was thinking of the root directory but thought maybe `services/gerichtsfinder/` is more explicit

File: app/services/gerichtsfinder/types.ts [hunk 2]
```diff
@@ -0,0 +1,98 @@
+export interface Jmtd14VTErwerberPlzortk {
```
[the remark this answers is not in the document]
Comment (chohner) [reply app/services/gerichtsfinder/types.ts:1]: probably, but I would wait until we are sure we wanna go with this approach :)

[the remark this answers is not in the document]
Comment (joschka) [reply app/services/gerichtsfinder/types.ts:1]: Hey, sorry, may I was to picky than. :wink:

File: app/services/gerichtsfinder/updateFromS3.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (joschka) [reply app/services/gerichtsfinder/updateFromS3.ts:1]: In my mind everything in app is bundled to one app. And services there are just the ones used in the app in run time.

**Pull request merged**